First Commit

cs-sujeet-ahirrao opened the pull request:
First Commit

cs-sujeet-ahirrao committed:
First Commit

First Commit

cs-pranali-chavan commented on widget/view.controller.js:
```diff
@@ -0,0 +1,107 @@
+        $scope.recordPlaybooks = _.filter(playbooks, item => item._hide === false);
+        $scope.recordPlaybooks = _.map($scope.recordPlaybooks, selectedPlaybook => {
+          let matchingPlaybook = _.find($scope.config.selectedPlaybooksWithRecord, item => item['@id'] === selectedPlaybook['@id']);
+          if (matchingPlaybook) {
```
add condition check for matchingPlaybook.icon also to prevent failure

cs-pranali-chavan commented on widget/widgetAssets/jsonToGrid.service.js:
```diff
@@ -0,0 +1,63 @@
+            installed: true
+          };
+          $http.post(API.BASE + 'widgets/detail', param).then(function (response) {
+            if (response.data['hydra:totalItems'] === 0) {
```
add check response.data['hydra:totalItems'] to handle case where response is empty []

cs-pranali-chavan reviewed (changes requested):
Overall code looks appropriate. few condition check to be added to avoid failure.

cs-sujeet-ahirrao committed:
BugFixes and Enhancement

- Incorporate review comments
- Generalize the widget i.e. using playbook we can launch the wizard or execute independentely

cs-pranali-chavan reviewed (approved):
Looks good
